### PR TITLE
chore: added facebook icon on navbar

### LINKED ISSUE #124: Add Facebook social media link to header
**Is your feature request related to a problem? Please describe.**
Allow users who are on the app find a community (Facebook).

**Describe the solution you'd like**
A button in the header with Facebook logo.
TODO: Ask for Facebook Page URL

**Describe alternatives you've considered**
In the future social links may change.

**Additional context**
Social Media Link Examples
image

**Technical ideas**
Modular design.

### PR DESCRIPTION
**Pull request by amelia2802**
Closes #124 

### Description of changes

- added facebook icon for dark/light mode on the navbar

### Additional context

-

### CODE REVIEW

**Review by keyserj — COMMENTED** [review]
just a small comment about the logo size.

also, can you squash/fixup that merge commit? i'm not too familiar with the merge flow for grabbing changes from the upstream repo, but if you use [this rebase flow](https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a) (using `main` instead of `master`), you shouldn't get a separate commit

File: web/src/common/components/Layout.tsx [hunk 1]
```diff
@@ -41,6 +44,9 @@ const Layout: NextPage<LayoutProps> = ({ children }) => {
               {/* <NavLink href="/mine">my understandings</NavLink> */}
               <NavLink href="/about">about</NavLink>
               {/* <NavLink href="/login">login</NavLink> */}
+              <NavLink href="https://www.facebook.com/profile.php?id=100091844721178" display="flex">
+                <Image src={facebookIconSrc} height={24} width={32} alt="facebook link" />
```
[the remark this answers is not in the document]
Comment (amelia2802) [reply web/src/common/components/Layout.tsx]: Sure @keyserj ! I've updated the style. You can check it now.

### updated preview
preview

**Review by keyserj — APPROVED** [review]
Nice, looks good
facebook-costume

P.S. I did some research on how to maintain a clean commit history when using `git merge` to keep your branch updated with the upstream repo - you may know this stuff already but I'm just gonna post my findings here:
- `git merge upstream/main` by default will [fast-forward](https://git-scm.com/docs/git-merge#Documentation/git-merge.txt---ff), i.e. get the latest commits from upstream/main by basing your branch off the latest commit on upstream/main
- using the `--no-ff` flag won't fast-forward, so your branch's base won't change, and commits from upstream have to be added via merge commit to your branch. this is slightly confusing for the PR because the merge commit has changes that were made in other PRs
- `git rebase upstream/main` will essentially do a fast-forward and cleans up the branch if it already has un-fast-forwarded merge commits on it
- GitHub's [squash and merge button](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits) will combine all the commits from a PR into one and essentially fast-forward away any merge commits. I'm going to just use this so you don't have to worry about cleaning up your commits - observing the history of popular GitHub repos makes it look like this is the most common practice used by maintainers
- the squash and merge button, since it combines all commits from a PR into one, won't be ideal for bigger PRs that make sense to have their changes split into multiple commits. not really a problem for this PR since it's smaller, but I'll need to think of what would be a good way of handling this 🤔

**Comment by keyserj** [comment]
thanks for contributing @amelia2802 🙂 we just added an [all-contributors section](https://github.com/amelioro/ameliorate#contributors-) to the README - can you let me know if you prefer _not_ to be recognized in this way?

@all-contributors please add @amelia2802 for code

**Comment by amelia2802** [comment]
Hi @keyserj, thank you for giving me the opportunity to contribute to this amazing project, also
 sharing the tips as I am unaware of the squash/merge button. It's really helpful.
And I'd like to add myself as a contributor and hope to make more useful contributions! 😊

**Pull request merged**